tests/pantable/test_auto_width.py

```diff
@@ -1,5 +1,6 @@
 """
 """
+from __future__ import division
```

Comment:
This probably is not needed. The reason is that I use `pasteurize` to the py2 wheel which add this automatically.